plugins/grpc/go/v1.5.0/buf.plugin.yaml

```
@@ -0,0 +1,22 @@
+version: v1
+name: buf.build/grpc/go
+plugin_version: v1.5.0
+source_url: https://github.com/grpc/grpc-go
+integration_guide_url: https://grpc.io/docs/languages/go/quickstart
+description: Generates Go client and server stubs for the gRPC framework.
+output_languages:
+  - go
+deps:
+  - plugin: buf.build/protocolbuffers/go:v1.34.2
+registry:
+  go:
+    min_version: "1.19"
+    deps:
+      - module: google.golang.org/grpc
+        version: v1.62.2
```

The useGenericStreams option defaults to true in this release, which requires pulling in 1.64.0 or later: https://github.com/grpc/grpc-go/blob/3eb01450ff91253bfc67a6b5b3c258ea2fc245c8/cmd/protoc-gen-go-grpc/grpc.go#L182. We should pull in v1.64.1 as it is a security release on top of 1.64.0.

```suggestion
        version: v1.64.1
```